modules/rosa-additional-principals-create.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-accessing-cluster.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-accessing-cluster.adoc
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-additional-principals-create_{context}"]
+= Adding additional principals while creating your ROSA cluster
```

Comment:
What about changing the title to ROSA HCP cluster? as ROSA classic cluster doesn't support this feature